// Copyright (C) 2020 THL A29 Limited, a Tencent company. All rights reserved.
//
// Licensed under the BSD 3-Clause License (the "License"); you may not use this
// file except in compliance with the License. You may obtain a copy of the
// License at
//
// https://opensource.org/licenses/BSD-3-Clause
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
// License for the specific language governing permissions and limitations under
// the License.

syntax = "proto3";

import "flare/rpc/rpc_options.proto";

option go_package = "tencent.com/protobuf/flare/example/protobuf_echo/relay_service_proto";
option cc_generic_services = true;

package example.protobuf_echo;

message RelayRequest {
  string body = 1;
}

message RelayResponse {
  string body = 2;
}

service RelayService {
  option (flare.qzone_service_id) = 2;
  rpc Relay(RelayRequest) returns (RelayResponse) {
    option (flare.qzone_method_id) = 1001;
  }
}
